Pipfile

```diff
@@ -14,6 +14,9 @@ flask = "~=1.1.4"
 flasgger = "~=0.9.5"
 matplotlib = "~=3.5.2"
 tabulate = "~=0.8.9"
+gdown = "*"
+yaml = "*"
+pyyaml = "*"
```

Comment:
We should specify some versions for these packages, I'd just take the newest ones and install with `pipenv install <package>~=<version>`